### ISSUE TITLE
SSL Environment Variables 

### ISSUE THREAD

**Issue by JulianBirch** [weavejester/ring-server#5]
Sorry, but I don't know quite where to put this one.  lein-ring documents that it supports the PORT and SSLPORT environment variables, but as far as I can tell, it doesn't.  This could be addressed at the lein-ring, ring-server or ring-jetty-adapter level.  I think ring-server is the right place.

If you agree, I'd be happy to stick together a pull request fixing this.

(Whilst we're on the subject, I'm not convinced PORT is a good name for an environment variable.)

**Comment by simon-brooke** [comment]
Browsing the code, I cannot see that SSL is supported at all. Am I missing something?

**Comment by weavejester** [comment]
@simon-brooke The options to standalone/serve are just passed straight through to the Jetty adapter, which does support SSL. However, it's more common to implement SSL via a proxy or load balancer, rather than on the adapter itself.

**Comment by JulianBirch** [comment]
Indeed. My original use case was to test https redirection code locally.
